frontend/src/components/GamePage.jsx

```diff
@@ -30,6 +41,7 @@ const GamePage = ({ gameMap, devMode = true }) => {
             show={showModal}
             onHide={() => setShowModal(false)}
         />
+        <ScorePhaseModal isOpen={showScoreModal} game={game} />
```

Comment:
I'm guilty of this too but it might be better for us to conditionally include the component instead of conditionally returning null, at least under normal circumstances.

```suggestion
        {showScoreModal && <ScorePhaseModal game={game} />}
```

https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null